Microsoft.VisualStudio.Terminal.Embeddable/Microsoft.VisualStudio.Terminal.Embeddable.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net46</TargetFramework>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <Version>0.0.0.0</Version>
+        <Authors>Daniel Griffen</Authors>
+        <Company />
+        <Description>Interop types for WhackWhackTerminal service</Description>
+        <Copyright>Copyright © Microsoft Corporation</Copyright>
+        <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/WhackWhackTerminal/master/LICENSE</PackageLicenseUrl>
+        <PackageProjectUrl>https://github.com/Microsoft/WhackWhackTerminal</PackageProjectUrl>
+        <PackageTags>WhackWhackTerminal VisualStudio Terminal</PackageTags>
+        <AssemblyVersion>0.0.0.0</AssemblyVersion>
```

Comment:
[Nerdbank.GitVersioning NuGet package](https://www.nuget.org/packages/Nerdbank.GitVersioning/) is an easy way to have project, vsix, and nuget package version be based on git height.